File: 151044081_hw7/DynamicArray.cpp
#include "DynamicArray.h"
#include <iostream>
#include <string>
#include <utility>

using namespace std;


template<class T>
void DynamicArray<T>::seta(T var){

	a = var ;
}

template<class T>
void DynamicArray<T>::setb(T var){

	b = var ;
}

template<class T>
void DynamicArray<T>::setvalue(int var){

	value = var;
}
template<class T>
T DynamicArray<T>::geta() const{

	return a;
}
template<class T>
T DynamicArray<T>::getb() const {

	return b;
}
template<class T>
int DynamicArray<T>::getvalue() const {

	return value; 
}

File: 151044081_hw8/DynamicArray.java
public class DynamicArray<T> {
    
    private T a ;
    
    private T b ;
    
    private int value = 0;
    
    public DynamicArray(){

        this.a = null;

        this.b = null;
        
    }
    
    public void seta(T var1){
        
        a = var1;
    }
    public void setb(T var1){
        
        b = var1;
    }
    public void setvalue(int var1){
        
        value = var1;
    }
    
    public T geta(){
        return a ;
    }
    
    public T getb(){
        return b;
    }
    public int getvalue(){
        
        return value;
    }
 
}
